packages/api/app/Models/User.js

```diff
@@ -24,6 +25,43 @@ class User extends Model {
 		});
 	}
 
+	static async create(payload) {
+		const modelInstance = new this();
```

Comment:
`new this()` fica bem estranho. Nesse caso podemos simplesmente fazer `new User()`;